docs/UserGuide.md

```diff
@@ -479,20 +479,21 @@ If your changes to the data file makes its format invalid, RecruitIn will discar
         * LEVEL_OF_EDUCATION inputs such as `miDDlE scHoOL` and `phD` are acceptable.
         * LEVEL_OF_EDUCATION inputs such as `Kindergarten` are not acceptable.
 * ##### YEARS_OF_EXPERIENCE `y/`
-    * A YEARS_OF_EXPERIENCE should be a **non-negative integer** smaller than or equals to **67** (re-employment age in Singapore).
+    * A YEARS_OF_EXPERIENCE should be a **non-negative number** with **at most 1 decimal place** and smaller than or equals to **67** (re-employment age in Singapore).
```

Comment:
Since we are only accepting intervals of `0.5`, this line should change to reflect that as well.

Reply:
Ok!